lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -1,4 +1,36 @@
 -- Propcore is allowed to everyone, but functions in the restrictedFunctions array will be restricted to devotee+ only
+local file_name = "cfc_propcore_whitelist"
+local whitelistTable = whitelistTable or {}
+
+if not file.Exists(file_name..".txt", "DATA") then
+    file.Write(file_name..".txt", "") --Creates new data file
```

Comment:
Why not just include .txt in `file_name` if its going to be appended every time?

Reply:
My thought on it was that people would forget to implement the `.txt` at the end... But I don't know I just thought it was better for the end user... /shrug